terraform/main.tf

```
@@ -143,9 +166,9 @@ resource "aws_lambda_function" "sgtm" {
   function_name    = "sgtm"
   role             = aws_iam_role.iam_for_lambda_function.arn
   handler          = "src.handler.handler"
-  source_code_hash = filebase64sha256("../build/function.zip")
+  source_code_hash = filebase64sha256("${data.archive_file.create_dist_pkg.output_path}")
```

could pull this expression out into a `local` to avoid computing it twice
```suggestion
  source_code_hash = filebase64sha256(data.archive_file.create_dist_pkg.output_path)
```